benchmark/fluid/fluid_benchmark.py

```diff
@@ -180,7 +180,7 @@ def train(avg_loss, infer_prog, optimizer, train_reader, test_reader, batch_acc,
         print_train_time(start_time, time.time(), num_samples)
         print("Pass: %d, Loss: %f" % (pass_id, np.mean(train_losses))),
         # evaluation
-        if not args.no_test and batch_acc:
+        if not args.no_test and batch_acc and not args.use_reader_op:
```

Comment:
CI didn't pass style check, please check if you have pre-commit installed properly.

Reply:
thank you for reminding.
have done